aten/src/ATen/native/cuda/Sort.cu

```diff
@@ -72,8 +376,9 @@ std::vector<int64_t> infer_dense_strides_dim_last(const Tensor & self, int64_t d
 // implementation here is a catch-all, so we're not looking for
 // efficiency, but instead correctness.
 std::tuple<Tensor &,Tensor &> sort_out_stable_cuda(const Tensor & self, c10::optional<bool> stable, int64_t dim, bool descending, Tensor & values, Tensor & indices) {
+  // use inplace algorithm for smaller input sizes
   if (should_use_th_sort(self, dim)) {
```

Comment:
sort is no longer in th, so you need to replace the name of `should_use_th_sort` with something else.